example-policies/basic.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: delivery.knative.dev/v1alpha1
```

Comment:
nit:  I think you can add comments here. Add a comment stating what the policy does in natural language for basic and advanced.

Reply:
fixed